security/compliance_operator/compliance-operator-supported-profiles.adoc

```diff
@@ -6,10 +6,15 @@ include::_attributes/common-attributes.adoc[]
 
 There are several profiles available as part of the Compliance Operator (CO) installation.
 
+[IMPORTANT]
+====
+The Compliance Operator might report incorrect results on managed platforms, such as OpenShift Dedicated, Red Hat OpenShift Service on AWS, and Azure Red Hat OpenShift. For more information, see the https://access.redhat.com/solutions/6983418[Red Hat Knowledgebase Solution #6983418].
```

Comment:
Should this link be included as an xref? Similarly how it is below?

Reply:
It's not an xref because it's a link to a solution article, not internal to content. :+1: